src/FixedDataTable.js

```diff
@@ -912,7 +912,7 @@ var FixedDataTable = React.createClass({
     }
 
     var lastScrollToRow  = oldState ? oldState.scrollToRow : undefined;
-    if (props.scrollToRow !== lastScrollToRow) {
+    if (typeof props.scrollToRow !== 'undefined' && props.scrollToRow !== null && props.scrollToRow !== lastScrollToRow) {
```

Comment:
I think we need to implement something similar for the other: scrollTop, scrollToRow and scrollToColumn